tests/integration/queries/test_dataset.py

```diff
@@ -78,6 +76,16 @@ def test_list_datasets(indico, airlines_dataset):
     assert type(datasets[0]) == Dataset
 
 
+def test_list_datasets_filtered(indico, airlines_dataset):
```

Comment:
might want to test a happy path as well, just to make sure the code doesn't filter out all datasets

Reply:
and by "a happy path" I mean a path where you do expect some datasets to be returned

Reply:
in the same func? i was thinking the existing `test_list_datasets` test above it kinda achieves that